scss/templates/user.scss

```diff
@@ -198,3 +148,188 @@
     }
   }
 }
+
+.stats-section {
+  .stats-title {
+    display: none;
+  }
+
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li,
+  .linked-stat {
+    margin: 0;
+    padding: 0;
+    margin: 0 1.5rem 1.5rem 0;
+
+    > a {
+      padding: 0 !important;
+    }
+  }
+
+  .user-stat {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 10rem;
+    height: 7.5rem;
+    background-color: $blue;
+    border-radius: 0.25rem;
+
+    .number,
+    .label,
+    .value {
+      color: $white;
+      font-style: normal;
+      font-weight: normal;
+    }
+
+    .number,
+    .value {
+      font-size: 2.25rem;
+      font-family: "Roboto", $font-family-sans-serif;
+    }
+
+    .label {
+      font-size: 0.75rem;
+      font-weight: bold;
+      text-transform: uppercase;
+      margin-top: 0.75rem;
+
+      .d-icon-heart {
+        display: none;
+      }
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    ul {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      grid-gap: 0.75rem;
+    }
+
+    li,
+    .linked-stat {
+      margin: 0;
+    }
+
+    .user-stat {
+      width: 100%;
+    }
+  }
+}
+
+.most-liked-section,
+.top-replies-section {
+  display: grid;
+  grid-gap: 2.4375rem;
+  grid-template-columns: 1fr;
+
+  .stats-title {
+    text-transform: none;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 1.125rem;
+    line-height: 1.375rem;
+    color: $gray-100;
+  }
+
+  .top-sub-section,
+  .summary-user-list {
+    width: 100%;
+    margin-bottom: 0;
+
+    > div > ul > li,
+    > ul > li {
+      height: auto;
+      border-bottom: 0.0625rem solid $gray-ligth;
+      border-left: none;
+
+      &:not(:last-child) {
+        margin: 0 0 0.78125rem;
+      }
+
+      &:last-child {
+        margin: 0;
+      }
+    }
+  }
+
+  &::before,
+  &::after {
+    display: none;
+  }
+
+  @include media-breakpoint-up(sm) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.most-liked-section .summary-user-list {
+  > div > ul > li {
+    .user-info {
+      margin-bottom: 0;
+
+      .user-image {
+        width: 2.5rem;
+        height: 2.5rem;
+
+        .user-image-inner a img {
+          width: 100%;
+          height: 100%;
+        }
+      }
+
+      .user-detail {
+        display: flex;
+        float: none;
+        height: 2.5rem;
+        justify-content: space-between;
+        align-items: center;
+        padding: 0;
+        width: 80%;
```

Comment:
_Suspicious..._ do you need the `80%` here?

Reply:
This is for using the maximum space available on a stat item, removing this rule we would have this

![image](https://user-images.githubusercontent.com/20747535/89696412-66a9d680-d8dd-11ea-8934-0849ee58c616.png)

Would I should add a comment for this line?

Reply:
Mmm maybe we should use paddings instead?

Reply:
@orlando not sure, by default, discourse has this style rule

![image](https://user -images.githubusercontent.com/20747535/89716016-707c1a00-d96f-11ea-84e8-b34be37ec9ae.png)

I only added a 10% so it looked a little bit wider

Let me know what you guys think

Reply:
We can afterwards revisit together. The idea here would be to have a flex container and define a width within the smaller component so the other items will _"naturally"_ fit within the whole container. But we can check it out together just to avoid the need of doing this in the future.

_For now we can move on and not stop the pull request due to this_